kobo/settings/base.py

```diff
@@ -871,6 +871,9 @@ def dj_stripe_request_callback_method():
     DJSTRIPE_WEBHOOK_VALIDATION = env.str('DJSTRIPE_WEBHOOK_VALIDATION', 'verify_signature')
 STRIPE_PUBLIC_KEY = STRIPE_LIVE_PUBLIC_KEY if STRIPE_LIVE_MODE else STRIPE_TEST_PUBLIC_KEY
 
+'''Organizations settings'''
```

Comment:
I would document this. We essentially would like to delete this some day. It would be helpful to explain that it's here to keep performance under control.